warden/src/test/kotlin/WardenTests.kt

```diff
@@ -1072,8 +1076,125 @@ class WardenTest : FreeSpec() {
             //TODO jailbroken iphone
         }
 
-        "And the Samsung" {
-            //TODO eternal leaves for samsung devices
+        "And the Fabulous" - {
+            val ios = "D4BD9EFC2A1AB1E2351143A4E67BB91F" to
+                    Json.decodeFromStream<Attestation>(WardenTest::class.java.classLoader.getResourceAsStream("ios-appattest.json"))
+            val android = "CAC4307080875C418BEB668E825649DC" to
+                    Json.decodeFromStream<Attestation>(this::class.java.classLoader.getResourceAsStream("aksattest.json"))
+
+            withData(ios, android) {
+                Warden(
+                    AndroidAttestationConfiguration.Builder(
+                        AndroidAttestationConfiguration.AppData(
+                            "at.asitplus.cryptotest.androidApp",
+                            listOf(
+                                "941A4513A3027563D3A6EA48EEE85BA45EB9F69CEEA19EF0EBB17F100BFC8878".hexToByteArray(
```

Comment:
Maybe extract to variable named `correctSigningCertificate`? or the whole configuration object?